shaker/salt_shaker.py

```diff
@@ -63,7 +65,9 @@ def __init__(self, root_dir, salt_root_path='vendor',
             clone_path(string): The directory to put formula into
             salt_root(string): The directory to link formula into
         """
-
+        # Run sanity checks on pygit2
+        if not pygit2_utils.pygit2_check():
+            sys.exit(1)
```

Comment:
Given that salt-shaker can also be called by python we should throw an exception, and not do sys.exit here.

We can make it a nice exception and catch it in our script to avoid messy backtraces, but doing sys.exit in a library is considered bad form.